### PR TITLE
Approve transactions

### PR DESCRIPTION
**Pull request by iVuong**

### CODE REVIEW

**Comment by thatniggafacts** [comment]
Let gets this to work

File: banking_system/transaction_management/templates/transaction_management/pendingCashierChecks.html [hunk 1]
```diff
@@ -55,7 +55,7 @@
 <script src="https://code.jquery.com/jquery-3.1.0.js"></script>
 <script>
 (function test(){
-    var statusMap = {"Approve": "APPROVED","Reject": "REJECTD"};
+    var statusMap = {"Approve": "APPROVED","Reject": "REJECTED"};
     console.log("hi")
```
Comment (zlmonroe) [inline banking_system/transaction_management/templates/transaction_management/pendingCashierChecks.html]: remove this :-P

File: banking_system/transaction_management/templates/transaction_management/pendingFundTransfers.html [hunk 2]
```diff
@@ -55,7 +55,7 @@
 <script src="https://code.jquery.com/jquery-3.1.0.js"></script>
 <script>
 (function test(){
-    var statusMap = {"Approve": "APPROVED","Reject": "REJECTD"};
+    var statusMap = {"Approve": "APPROVED","Reject": "REJECTED"};
     console.log("hi")
```
Comment (zlmonroe) [inline banking_system/transaction_management/templates/transaction_management/pendingFundTransfers.html]: Remove this too :-P

**Review by zlmonroe — DISMISSED** [review]
Just two log lines. I'm going to wait on approval until you look over mine though

**Review by zlmonroe — APPROVED** [review]
This is good. There is an earlier problem from credit where it asked for an email we should fix.

**Pull request merged**